apps/gallery/style/gallery.css

```diff
@@ -851,6 +851,7 @@ a.filter-faded {
   height: 2.6rem;
   -moz-margin-start: calc(3.9rem - 1.8rem);
   background: url(images/slider-scrubber.png) no-repeat center center;
+  background-size: 100% auto;
```

Comment:
Updated patch to include backgound-size in the same line by specifying 100%.  Lack of 100% width dimension to background-size was causing big background image to get truncated in the containing box. 
I will file a bug to look into images picked at 2.25 device pixel ratio for Aries where Aries window.devicepixelratio is  2.